pattern-library/sass/patterns/_grid.scss

```diff
@@ -14,276 +14,180 @@
-$grid-direction-default: ltr;
-$grid-direction-reversed: rtl;
-
-@if $layout-direction == rtl {
-    $grid-direction-default: rtl;
-    $grid-direction-reversed: ltr;
+// Utility functions/mixins designed for Susy API-compatibility.
+@function cols($cols) {
+    @return $cols / $grid-column-count * 100%;
 }
 
-// susy-based configuration (see http://susydocs.oddbird.net/en/latest/settings/)
-$susy: (
-    use-custom: (
-        background-options: false,
-        box-sizing: false,
-        breakpoint: true,
-        clearfix: true,
-        rem: true
-    ),
-    debug: (
-        color: $grid-debug-color,
-    ),
-    math: fluid,
-    output: float
-);
-
-// LAYOUT: set default susy settings
-// (gutters are handled outside of elements - by default)
-$susy-default: (
-    flow: $grid-direction-default,
-    column-width: $grid-column-width,
-    columns: $grid-column-count,
-    container-position: $grid-container-position,
-    container: $grid-max-width,
-    global-box-sizing: border-box,
-    gutters: $grid-gutter-width,
-    gutter-position: $grid-gutter-position
-);
-
-// LAYOUT: set susy settings especially for manually set grids in the DOM
-// (gutters are handled inside of elements - by default)
-$susy-manual: layout($susy-default $grid-manual-gutter-position);
-
-// LAYOUT: set debug settings
-$susy-debug: layout($susy-default show);
-
-
-// ----------------------------
-// #GLOBALS
-// ----------------------------
-// set basic grid settings for app
-@include layout($susy-default);
+@mixin span($cols, $margin:'') {
+    @include margin-left(calc(0.5 * #{$grid-gutter-width}));
+    @include margin-right(calc(0.5 * #{$grid-gutter-width}));
+    width: calc(#{cols($cols)} - #{$grid-gutter-width});
+
+    @if ($margin == 'before') {
+        @include margin-right(0);
+        width: calc(#{cols($cols)} - 0.5 * #{$grid-gutter-width});
+    } @else if ($margin == 'after') {
+        @include margin-left(0);
+        width: calc(#{cols($cols)} - 0.5 * #{$grid-gutter-width});
+    } @else if ($margin == 'none') {
+        @include margin-left(0);
+        @include margin-right(0);
+        width: cols($cols);
+    }
+}
 
+@mixin pre($cols) {
+    @include margin-left(cols($cols) !important);
+}
 
-// ----------------------------
-// #UTILITY
-// ----------------------------
-%grid-container {
-    @include container();
+@mixin post($cols) {
+    @include margin-right(cols($cols) !important);
 }
 
 // grid rows - set visual break and document flow clear (clearfix)
-%grid-row {
-    @include break();
-    @include clearfix();
+@mixin grid-container($direction:$grid-direction, $size:$grid-max-width) {
+    display: flex;
+    max-width: $size;
+    flex-direction: $direction;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: flex-start;
+    box-sizing: border-box;
+    margin: 0 auto;
 }
 
-%grid-gallery {
-    @include break();
-    @include clearfix();
+@mixin grid-row {
+    @include span(12, none);
```

Comment:
Yep, this one needs to be a variable - it should fill the whole row regardless of what you have  specified as your default number of grid rows, so I'll use that variable here.